actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -139,6 +141,31 @@ def negotiate_mime(order)
         order.include?(Mime::ALL) ? format : nil
       end
 
+      class VariantInquirer # :nodoc:
+        delegate :==, :each, :empty?, to: :@variants
```

Comment:
Perhaps `==` should be delegated to `@variants` in globbed form:

```ruby
def ==(*variants)
  @variants == variants
end
```

Before:
`request.variant == :tablet # will always return false so probably not something user wants`

After:
`request.variant == :tablet` # will return true if tablet is the exact match and no other variants exist.